.github/workflows/ci_tests.yml

```diff
@@ -16,13 +16,14 @@ jobs:
     strategy:
       matrix:
         preset: ["gcc-debug", "gcc-release"]
+    name: "Preset Test: ${{ matrix.preset }}"
     steps:
       - uses: actions/checkout@v4
-      - name: Setup cmake
-        uses: jwlawson/actions-setup-cmake@v2
+      - name: Setup build environment
+        uses: lukka/get-cmake@latest
         with:
-          cmake-version: '3.25.x'
-      - uses: seanmiddleditch/gha-setup-ninja@v5
+          cmakeVersion: "~3.25.0"
+          ninjaVersion: "^1.11.1"
```

Comment:
Just tidying stuff up here. Giving preset test a better name and use the same setup-cmake action as the one below for consistency.